cura/OAuth2/SecretStorage.py

```diff
@@ -0,0 +1,20 @@
+import keyring
+
+
+class SecretStorage:
```

Comment:
Missing documentation. What does this class do, What is it's point? How should it be used?

Reply:
You jumped the gun here @nallath ;-) Still very much in the "in progress" column at the moment.
Anyhow. I removed the SecretStorage class since I now Use a descriptor for the authenicationresponsemodel attributes